spec/data/sample_manifest_excel/columns.yml

```diff
@@ -741,8 +741,8 @@ disease:
   unlocked: true
   conditional_formattings:
     empty_cell:
-donor_id2:
-  name: :donor_id
+legacy_donor_id:
```

Comment:
I renamed this to make it clearer what it is. Plus I have a horrible feeling that we've probably been inserting this heading into our manifests, rather than the correct one.